.drone.yml

```diff
@@ -96,6 +96,7 @@ steps:
 - name: elixir-test
   image: prima/amqpx-ci:${DRONE_COMMIT}
   commands:
+  - sleep 15
```

Comment:
why?

Reply:
I feel embarassed by this, but anyway: the CI is so fast that it gets to the `mix test` step before the rabbit containers are up and running. Adding one additional rabbit container made this problem more evident as several CI runs failed in a row...

Reply:
can't we try and add a command to check if they are up instead of waiting for a random amount of time? like a telnet or something... :-(
Didn't we had something like that in one of our projects?

Reply:
I don't know how to do that with rabbit, also if you look at the CI logs of the container, it has a weird lifecycle: it starts, then stops, then starts again and only then it's alive.
wrt other projects having this check, i only know about ping to mysql containers when developing locally, i've never heard of something similar in CI... usually no project has a CI so fast :D

Reply:
> can't we try and add a command to check if they are up instead of waiting for a random amount of time? like a telnet or something... :-( Didn't we had something like that in one of our projects?

Maybe, btw in test_helper.exs you could check if the connectionManager return a valid connection before running al tests...

Reply:
> > can't we try and add a command to check if they are up instead of waiting for a random amount of time? like a telnet or something... :-( Didn't we had something like that in one of our projects?
> 
> Maybe, btw in test_helper.exs you could check if the connectionManager return a valid connection before running al tests...

I like it